forwarder/kernel-forwarder/pkg/kernelforwarder/local/local.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Please keep the license of the code when you move it to new files. Having two Copyrights is completely fine in such a case when you introduce significant changes too.

Reply:
Done